src/Events/Migration/v0.14/01-setup-functions.sql

```diff
@@ -0,0 +1,32 @@
+﻿CREATE OR REPLACE FUNCTION events.getsubscriptions(
+	source character varying,
+	subject character varying,
+	type character varying)
+    RETURNS TABLE(
+		id bigint,
+		sourcefilter character varying, 
+		subjectfilter character varying, 
+		typefilter character varying, 
+		consumer character varying, 
+		endpointurl character varying, 
+		createdby character varying, 
+		validated boolean, 
+		"time" timestamp with time zone) 
+    LANGUAGE 'plpgsql'
+    COST 100
+    VOLATILE PARALLEL UNSAFE
+    ROWS 1000
+
+AS $BODY$
+
+BEGIN
+return query 
+	SELECT s.id, s.sourcefilter, s.subjectfilter, s.typefilter, s.consumer, s.endpointurl, s.createdby, s.validated, s."time"
+	FROM events.subscription s
+  WHERE (s.subjectfilter is NULL OR s.subjectfilter = subject)
+  AND position(s.sourcefilter in source) = 1
```

Comment:
I was wrong when I said that we should implement `position(subscription.source_filter in event.source) = 1` before we had match-format with brackets.
We should change this (back) to `s.sourcefilter = source` until #96 .

The problem is that without match-format we will incorrectly match
subscription source-filter = `apps.altinn.cloud/ttd/appname`
with an event from source = `apps.altinn.cloud/ttd/appname-print`.

If we reverse the parameter ordering we will match
subscription source-filter = `apps.altinn.cloud/ttd/appname-print`
with an event from source = `apps.altinn.cloud/ttd/appname`.

So, until #96 we should just keep it as an strict `=` since we have build the source explicitly to have 2 path-segments.